public/docs/_examples/structural-directives/dart/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: structural_directives
+description: Structural directives example
```

Comment:
Have you considered adding https://www.dartlang.org/tools/pub/pubspec.html#sdk-constraints?